doc/tutorial/single_cell_detailed_recipe.rst

```diff
@@ -59,18 +49,18 @@ examine the recipe in detail: how to create one, and why it is needed.
 
 .. literalinclude:: ../../python/example/single_cell_detailed_recipe.py
    :language: python
-   :lines: 91-139
+   :lines: 84-133
 
 Let's go through the recipe point by point.
 
-Step **(6)** creates a ``single_recipe`` class that inherits from :class:`arbor.recipe`. The base recipe
+Step **(5)** creates a ``single_recipe`` class that inherits from :class:`arbor.recipe`. The base recipe
 implements all the methods defined above with default values except :meth:`arbor.recipe.num_cells`,
 :meth:`arbor.recipe.cell_kind` and :meth:`arbor.recipe.cell_description` which always have to be implemented
 by the user. The :meth:`arbor.recipe.global_properties` also needs to be implemented for
 :class:`arbor.cell_kind.cable` cells. The inherited recipe can implement any number of additional methods and
 have any number of instance or class variables.
 
-Step **(6.1)** defines the class constructor. In this case, we pass a ``cell`` and a set of ``probes`` as
+Step **(5.1)** defines the class constructor. In this case, we pass a ``cell`` and a set of ``probes`` as
```

Comment:
Note below as well, no need to send `probes` around. Do you know why this was done?

Reply:
I don't know why this was done. Maybe to be consistent with the pattern we use for the cell description? That happens in other tutorials this way too. It would be nicer perhaps to move the cell description into a function, or simply into the `cell_description` method. Is that what you mean for the probes as well?